CREATE_BRIDGE.yml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
```

Comment:
I thought it should be the network operator the one that creates it?

Reply:
Don't look! It is just testing that this approach would work. It will become a part of kubevirtci.

We don't have node network configuration implemented (yet). Once, it's done. This workaround will be dropped from kubevirtci containers and replaced with proper components, as we did with CNI and multus before.